api/applications.go

```diff
@@ -228,6 +229,7 @@ func ApplicationEnvironmentModuleUpdate(accountID int, applicationID int, env st
 	}
 	log.Printf("[DEBUG] JSON payload: %s\n", b)
 	headers := map[string][]string{"filepath": []string{filePath}}
+	Timeout = 300 * time.Second // five minutes, because server-side validation can take a while
 	resp, err := request(http.MethodPatch, u, bytes.NewBuffer(b), headers)
```

Comment:
Shouldn't `request` accept `context.Context` with timeout?

Reply:
Yeah, you're right, my implementation here is crappy, and I should be using `context.Context`.

I'll refactor this now.